Build errors from 'tsc -p ./' aren't displayed in CI

We had a build failure where `gulp package` displayed this:
```
[13:11:30] Using gulpfile ~/Repos/vscode-cosmosdb/gulpfile.js
[13:11:30] Starting 'package'...
Executing prepublish script 'npm run vscode:prepublish'...
[13:11:36] 'package' errored after 5.54 s
[13:11:36] Error: Command failed: npm run vscode:prepublish
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! vscode-cosmosdb@0.4.0 vscode:prepublish: `tsc -p ./`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the vscode-cosmosdb@0.4.0 vscode:prepublish script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/ericjizba/.npm/_logs/2018-03-15T20_11_36_449Z-debug.log

    at formatError (/usr/local/lib/node_modules/gulp/bin/gulp.js:169:10)
    at Gulp.<anonymous> (/usr/local/lib/node_modules/gulp/bin/gulp.js:195:15)
    at emitOne (events.js:115:13)
    at Gulp.emit (events.js:210:7)
    at Gulp.Orchestrator._emitTaskDone (/Users/ericjizba/Repos/vscode-cosmosdb/node_modules/orchestrator/index.js:264:8)
    at /Users/ericjizba/Repos/vscode-cosmosdb/node_modules/orchestrator/index.js:275:23
    at finish (/Users/ericjizba/Repos/vscode-cosmosdb/node_modules/orchestrator/lib/runTask.js:21:8)
    at /Users/ericjizba/Repos/vscode-cosmosdb/node_modules/orchestrator/lib/runTask.js:45:4
    at <anonymous>
```

But if I ran `tsc -p ./` locally, I saw this:
```
src/graph/GraphViewServer.ts(44,20): error TS2503: Cannot find namespace 'SocketIO'.
src/graph/GraphViewServerSocket.ts(10,32): error TS2503: Cannot find namespace 'SocketIO'.
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! vscode-cosmosdb@0.4.0 vscode:prepublish: `tsc -p ./`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the vscode-cosmosdb@0.4.0 vscode:prepublish script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/ericjizba/.npm/_logs/2018-03-15T20_11_56_374Z-debug.log
```

The second log is much more helpful and has errors and files that need to be fixed

--- a/.travis.yml
+++ b/.travis.yml
@@ -16,6 +16,7 @@ install:
   - npm install
 
 script:
+  - npm run build
   - gulp package
   - gulp upload-vsix
   - npm run lint

